lib/travis/yaml/web/config.rb

```diff
@@ -0,0 +1,7 @@
+require 'travis/config'
+
+module Travis::Yaml::Web
+  class Config < Travis::Config
+    define auth_keys: ['abc123']
```

Comment:
Config for `auth_keys` has been added to keychain and trvs-uploaded to (for the moment) the org-staging app so am thinking this needs to change? Or is this just setting up a placeholder for local/dev work...?

Reply:
That only gets loaded if the keychain version isn't found. 👍